### ISSUE TITLE
Missing "Options" menu = missing intro subtitles

### ISSUE THREAD

**Issue by glaucon1984** [alexbatalov/fallout1-ce#36]
There is no option menu (at least in the x64 Linux and Windows versions), this means you can only change the options once you start the game.
As subtitles are disabled by default, there is no way to be able to watch the intro or the first overseer cinematic with subtitles, which could be important for hear-impaired players and non-native English speakers trying to play a translated version of the game.
If the Options menu cannot be added on the Main Menu, it would be nice to provide some instructions on how to enable subtitles through a config file. Or just make subtitles enabled by default.

**Comment by Vic2077** [comment]
This is the problem of the original game You can watch intro with subtitles by starting a new game, enabling subtitles, then exiting the game and starting it over. Or bring your old cfg file and add it to the game files that ypou are importing into this app. This has nothing to do with this app or its author, however, I agree that adding Opotions button to the Main menu (if possible) would be great. I don't have hearing problems, although I also like using subtitles and also prefer to change many other options in Fallout 1 and 2, so I also would be really gald to see Options menu shortcut in the Main Menu of the game. It always looks as if it's a bug, because there even is a background plate in the main menu for it, and High Resolution patch brings the Options shortcut back:
![Screenshot Vanilla 1 1](https://user-images.githubusercontent.com/22431284/220245774-0e32035e-eea3-43a7-b2ed-8f6ab6f5253f.png)
![Screenshot HiResPatch 1 2](https://user-images.githubusercontent.com/22431284/220245803-edcfd299-f34c-4b6a-bf6d-81108963ca9f.png)

**Comment by emxd** [comment]
@Vic2077 

Are subs working for you? I tried both release x86,x64 and CI/CD debug x86,x64 and they weren't displaying for me. Win11 GOG. Worked in fallout1-re.

**Comment by glaucon1984** [comment]
Unlike the original game, fallout-ce won't have subtitles after they have been enabled in a new game, exiting the game and starting the intro or another new game.

**Comment by Vic2077** [comment]
> @Vic2077
> 
> Are subs working for you? I tried both release x86,x64 and CI/CD debug x86,x64 and they weren't displaying for me. Win11 GOG. Worked in fallout1-re.

They do. I'm on Win7x64 (I have many different versions, including GOG, STEAM, old Russian CDs from early 2000s - mostly with Fargus localisation and a copy of the original Eng CD version (though it's not an original Eng CD, but an image of that CD burned onto a CD-R disc).

**Comment by glaucon1984** [comment]
> They do

Can you watch the intro with subtitles? And the initial overseer speech? I just tested it and it doesn't work for me.

**Comment by Vic2077** [comment]
> > They do
> 
> Can you watch the intro with subtitles? And the initial overseer speech? I just tested it and it doesn't work for me.

Yes. Although, the screenshots don't capture the video for some rerason.
![scr00002](https://user-images.githubusercontent.com/22431284/221351369-f1e71d32-b312-4ff9-acbd-fb78aaccddd3.png)
![scr00003](https://user-images.githubusercontent.com/22431284/221351370-52fd9872-95bd-4dd4-9640-42984c584b07.png)
![scr00004](https://user-images.githubusercontent.com/22431284/221351371-a1b6fb3f-8fda-4d8b-98e4-2deb0c1852e0.png)

**Comment by emxd** [comment]
@Vic2077

And what distribution is this specifically?

**Comment by c6-dev** [comment]
Just tried the latest x64 GHA build (Windows 11, Steam) and i am not seeing subtitles either.

**Comment by Vic2077** [comment]
> 

Latest available on GOG + Fallout1-ce 1.0 (release)

**Comment by alexbatalov** [comment]
You can enable `subtitles` manually by editing `fallout.cfg`. If you don't want to mess with it - start a new game, enable subtitles in preferences, exit to main menu, press Intro to see it with subtitles, start a new game once again (this time with subtitles).

I've checked original menu background from v1.1 and v1.2 - there is no empty section for "Options" button. I guess it is also part of HRP for F1.

**Comment by Distur1ed** [comment]
hello how add Configuration Fallout 1 -> Settings in Amiga

<img width="2880" height="2160" alt="Image" src="https://github.com/user-attachments/assets/3c0dc41b-7bc4-4331-a8e2-ffe4d7989612" />
